atlas/AtlasMl/atlasml/ml/MLPipelines/PipelineWorkflows.py

```
@@ -163,29 +162,32 @@ def initial_cluster_pipeline(self, eps: float = 0.1, min_samples: int = 1, min_c
+                "exercise_id": exercise["properties"]["exercise_id"],
+                "description": exercise["properties"]["description"],
                 "competency_ids": [competency_id]
             }
-            self.weaviate_client.update_property_by_id(CollectionNames.TEXT.value, text_entry["id"], properties)
+            self.weaviate_client.update_property_by_id(CollectionNames.EXERCISE.value, exercise["id"], properties)
 
         return
 
 
-    def newTextPipeline(self, text: str, uuid: str):
+    def newTextPipeline(self, text: str) -> Competency:
```

Ruff (0.12.2)

Function name `newTextPipeline` should be lowercase

(N802)